src/appshell/view/dockwindow/dockpanel.cpp

```diff
@@ -158,6 +175,16 @@ void DockPanel::setClosable(bool closable)
     emit closableChanged(closable);
 }
 
+void DockPanel::setVisible(bool visible)
```

Comment:
Better to move this method (and all related ones) to DockView to prevent duplication of code

Reply:
thanks, fixed